tests/storage/snapshot.go

```diff
@@ -516,21 +508,16 @@ var _ = SIGDescribe("[Serial]VirtualMachineSnapshot Tests", func() {
 				tests.WaitAgentConnected(virtClient, vmi)
 
 				createDenyVolumeSnapshotCreateWebhook()
+				defer deleteWebhook()
```

Comment:
isn't it the redundant as it happens in the AfterTest?

Reply:
wasn't happening in my testing